src/modules/form/components/group/Signature.tsx

```diff
@@ -21,25 +27,25 @@ const Signature = ({ item, renderChildItem }: GroupItemComponentProps) => {
       }}
     >
       <fieldset style={{ border: 'none', margin: 'none', padding: 'none' }}>
-        {item.text && (
-          <legend>
-            <Typography
-              variant='h5'
-              color={theme.palette.primary.main}
-              sx={{ mb: 2 }}
-            >
-              {item.text}
+        <Stack gap={2}>
```

Comment:
Added stack since certification should only be shown if `viewOnly==false`, and we dont want to keep the extra padding below the children. This is why I usually go for Stack rather than adding `sx={{ mb: 2 }}` between components

Reply:
great!